views/export3.jade

```diff
@@ -0,0 +1,36 @@
+extends layout
+
+block content
+    include topnav
+    h1 Step 3: Preview
+    each songs in songss
```

Comment:
i'd probably pick a different variable name than songss since songss to me is easy to confuse with songs